setup.py

```diff
@@ -20,10 +20,6 @@
     classifiers=[
         'License :: Appache-2.0'
     ],
-    install_requires=["Qt.py",
-                      "blinker",
-                      "nine",
-                      "docutils",
-                      ],
+    install_requires=["enum", "Qt.py", "blinker", "nine", "docutils"],
```

Comment:
What about python versions 3.4+ where we dont need enum ? since it is built-in

Reply:
You are right. Coming soon ;)

Reply:
- [x] done, tested with python2 and python3